CodeStarterCamp_Week4/Routine.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Routine.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by 홍승주 on 2022/05/05.
+//
+
+import Foundation
+
+struct Routine {
+    let routineName: String
+    var exercises: Array<Exercise>
+    
+    func startRoutine() {
+        print("--------------")
+        print("hellRoutine을 시작합니다.")
```

Comment:
이부분은 routineName을 출력하는 것 같아요!! 개선해볼까요?

Reply:
startRoutine의 네이밍을 개선하면 되는 건가요...??

Reply:
```suggestion
        print("\(routineName)을 시작합니다.")
```
이러한 로직으로 구성될수도 있다는 말이었어요!! 이부분은 크게 중요하지 않으니 넘어갑시당~